packages/react/src/formElements/MdSelect.tsx

```diff
@@ -66,6 +66,31 @@ const MdSelect: React.FunctionComponent<MdSelectProps> = ({
     displayValue = '';
   }
 
+  useEffect(() => {
+    document.addEventListener("keydown", onKeyDown, false);
+
+    return () => {
+      document.removeEventListener("keydown", onKeyDown, false);
+    }
+  });
+
+  const onKeyDown = (e: any) => {
```

Comment:
Kunne kanskje utvidet til å ta imot piltast opp og ned for å kunne gå opp og ned i listen også uten å bruke musepekeren? (Ikke must have akkurat nå)